language-support/ts/codegen/BUILD.bazel

```diff
@@ -1,12 +1,36 @@
 # Copyright (c) 2020 The DAML Authors. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-load("//bazel_tools:haskell.bzl", "da_haskell_binary")
+load("//bazel_tools:haskell.bzl", "da_haskell_binary", "da_haskell_library")
 load("//bazel_tools/packaging:packaging.bzl", "package_app")
 
+da_haskell_library(
+    name = "daml2ts-helpers",
+    srcs = glob(["src/DA/Daml/*.hs"]),
+    hackage_deps = [
+        "aeson",
+        "base",
+        "bytestring",
+        "filepath",
+        "hashable",
+        "text",
+        "unordered-containers",
+    ],
+    src_strip_prefix = "src",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//libs-haskell/da-hs-base",
+    ],
+)
+
 da_haskell_binary(
     name = "daml2ts",
-    srcs = glob(["src/**/*.hs"]),
+    srcs = glob(
+        ["src/**/*.hs"],
+        exclude = [
+            "src/DA/**/*.hs",
```

Comment:
Pretty sure that is not going to work on Windows as you expect. Due to the lack of sandboxing you end up pulling in `daml2ts-helpers` as sources instead of from the library. You need to put them in separate directories

Reply:
Thanks for pointing this out. I hope https://github.com/digital-asset/daml/pull/5214 does a better job?